sp.yml

```diff
@@ -0,0 +1,39 @@
+---
+- hosts: "{{ target | default('localhost') }}"
+  become: no
+  connection: ssh
+  gather_facts: true
+
+  vars_files:
+    - "inventory/host_vars/all"
+    - "inventory/group_vars/all"
+    - "inventory/group_vars/{{ env }}/vars.yml"
+    - "inventory/group_vars/{{ env }}/vault.yml"
+    - "inventory/group_vars/{{ pop }}/vars.yml"
+    - "inventory/group_vars/{{ pop }}/vault.yml"
+    - "inventory/group_vars/{{ plat }}/vars.yml"
+    - "inventory/group_vars/{{ plat }}/vault.yml"
+
+  pre_tasks:
+    - name: get my public IP
```

Comment:
Not sure the function of this pre_task. Can we remove it? The IP addresses are fixed in the environment file.

Reply:
absolutely: to be removed because it is useless